project/Dependencies.scala

```diff
@@ -18,7 +18,7 @@ object Dependencies {
   val slf4jVersion = "1.7.22"
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.11.8", "2.12.1"),
+    crossScalaVersions := Seq("2.12.1"),
```

Comment:
it's in the tmp commit that I will remove when cleaning up this. I ran bench with 2.12